File: app/src/main/java/android/montreal/devfest/app/Application.java
package android.montreal.devfest.app;

import android.montreal.devfest.stuff.policies.DaggerSuffPoliciesComponent;

import dagger.android.AndroidInjector;
import dagger.android.support.DaggerApplication;

public class Application extends DaggerApplication {
    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerApplicationComponent.builder()
                .with(this)
                .with(DaggerSuffPoliciesComponent.create())
                .build();
    }
}

File: app/src/main/java/android/montreal/devfest/app/Application.kt
package android.montreal.devfest.app

import android.montreal.devfest.stuff.policies.DaggerSuffPoliciesComponent

import dagger.android.AndroidInjector
import dagger.android.support.DaggerApplication

class Application : DaggerApplication() {
    override fun applicationInjector(): AndroidInjector<out DaggerApplication> {
        return DaggerApplicationComponent.builder()
                .with(this)
                .with(DaggerSuffPoliciesComponent.create())
                .build()
    }
}
